pkg/virt-config/configuration_test.go

```diff
@@ -661,5 +661,8 @@ var _ = Describe("test configuration", func() {
 			Expect(clusterConfig.LiveMigrationEnabled()).To(BeTrue())
 		})
 
+		It("SR-IOV live migration feature gate", func() {
+			Expect(clusterConfig.SRIOVLiveMigrationEnabled()).To(BeTrue())
```

Comment:
What is the reason for keeping `clusterConfig.SRIOVLiveMigrationEnabled()` and not removing it?

Reply:
Hey @AlonaKaplan :)
According to our new [deprecation policy](https://github.com/kubevirt/kubevirt/blob/main/docs/deprecation.md), the feature gates need to remain as no-ops. That's also the behavior implemented in [the PR](https://github.com/kubevirt/kubevirt/pull/8047) for deprecating live migration feature gate.

So, IMHO, this should stay

Reply:
It's not really necessary we could use `config.isFeatureGateEnabled()` directly.
I left it there just for consistency as a similar method of LiveMigration feature gate is still there.
Do you prefer it to be removed?

Reply:
Somehow I saw cache version of this page when commenting :shrug: